scrapy/loader/__init__.py

```diff
@@ -3,219 +3,104 @@
 
 See documentation in docs/topics/loaders.rst
 """
-from collections import defaultdict
-from contextlib import suppress
+import itemloaders
 
 from scrapy.item import Item
-from scrapy.loader.common import wrap_loader_context
-from scrapy.loader.processors import Identity
 from scrapy.selector import Selector
-from scrapy.utils.misc import arg_to_iter, extract_regex
-from scrapy.utils.python import flatten
 
 
-def unbound_method(method):
+class ItemLoader(itemloaders.ItemLoader):
     """
-    Allow to use single-argument functions as input or output processors
-    (no need to define an unused first 'self' argument)
-    """
-    with suppress(AttributeError):
-        if '.' not in method.__qualname__:
-            return method.__func__
-    return method
+    Return a new Item Loader for populating the given Item. If no item is
```

Comment:
It seems this docstring explains how `ItemLoader.__init__` works, not what is ItemLoader class about. Could it be moved to `__init__` method (is autodocs output reasonable?)?

Alternatively, it can be rewritten as a class docstring. This makes more sense to me, because e.g. attribute docs don't really belong to `__init__` docstring.